component/Bubbles.js

```diff
@@ -8,6 +8,7 @@ function Bubbles(container, self, options) {
   sidePadding = options.sidePadding || 6 // padding on both sides of chat bubbles
   recallInteractions = options.recallInteractions || 0 // number of interactions to be remembered and brought back upon restart
   inputCallbackFn = options.inputCallbackFn || false // should we display an input field?
+  bubbleCallbackFn = options.bubbleCallbackFn || false // is there a callback function for when a user clicks on a bubble button
```

Comment:
Could we name this `responseCallbackFn` to match `inputCallbackFn` naming a little closer?